src/GNDStk/type2string/test/type2string.test.cpp

```diff
@@ -0,0 +1,9 @@
+
+#define CATCH_CONFIG_MAIN
+
+#include "catch.hpp"
+#include "GNDStk.hpp"
+
+SCENARIO("Testing GNDStk convert(type,string)") {
+
```

Comment:
Looks like there are several things that are not tested. How do we know they work?

Reply:
As you noticed, several of the test codes are just stubbed for now. Most (in some cases all) of the material they'd test are in fact tested indirectly elsewhere, as a consequence of testing other things for which these are dependencies. I hadn't finished these because, given the circumstances, other things were arguably more important. However, perhaps it's time to go in and fill out the various test files that are stubs right now.